library/Zend/Mvc/Service/ServiceListenerFactory.php

```diff
@@ -53,6 +53,7 @@ class ServiceListenerFactory implements FactoryInterface
             'DiStrictAbstractServiceFactory' => 'Zend\Mvc\Service\DiStrictAbstractServiceFactoryFactory',
             'FilterManager'                  => 'Zend\Mvc\Service\FilterManagerFactory',
             'FormElementManager'             => 'Zend\Mvc\Service\FormElementManagerFactory',
+            'Zend\Form\Annotation\FormAnnotationBuilder' => 'Zend\Mvc\Service\FormAnnotationBuilderFactory',
```

Comment:
On merge, I'll add an alias for the FQCN; for v3, we can move to all FQCN, but considering no other framework-defined factories do this currently, we should remain consistent.